### PR TITLE
Update the L3 HA document in the Howtos section

### PR DESCRIPTION
**Pull request by PioMis1984**

### CODE REVIEW

**Comment by PioMis1984** [comment]
I've added tabs: one for legacy regions and the second for new regions. There are only a minor differences between the two.
Also added a warning message at the top with our recommendation to use LBaaS instead.

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ---
 title: 'Building Layer 3 High Availability'
 published: true
-date: '16-09-2018 16:20'
+date: '29-03-2024 10:20'
 taxonomy:
     category:
         - docs
 ---
+! This solution is deprecated, we highly recommend to use Octavia LBaaS for this purpose: [Create a Load Balancer](../../02.Tutorials/05.lbaas/docs.en.md)
```
Comment (matthiasbernhardt) [inline user/pages/03.Howtos/10.l3-high-availability/docs.en.md:9]: This approach is not deprecated, it is perfectly valid for certain use cases but can be tedious compared to LBaaS, so using Octavia LBaaS should at least be considered before opting for this variant.

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md [hunk 2]
```diff
@@ -20,13 +21,21 @@ taxonomy:
 
 * `keepalived` is an open source VRRP implementation
 
-## Considerations
+## Legacy vs New regions
+
+Our legacy regions (cbk, dbl) have a different SDN backed than our new regions (fes, dus2).
```
Comment (matthiasbernhardt) [inline user/pages/03.Howtos/10.l3-high-availability/docs.en.md:26]: Legacy vs. new sounds negative / biased. Should we just write about differences neutrally?

**Review by matthiasbernhardt — COMMENTED** [review]
Only two small remarks on the distinction between our regions that may also apply to other places with wordings like "in cloud environments" which may neet to be reworded in both variants.

I think we do not cover the usage of floating IPs in conjunction with such virtual IPs yet. Perhaps we can add that or at least create a new ticket to do it separately.

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md [hunk 3]
```diff
@@ -20,13 +21,21 @@ taxonomy:
...
+Each SDN requires a dedicated `keepalived` configuration - both are presented below.
+
+[ui-tabs position="top-left" active="0" theme="lite"]
+[ui-tab title="Legacy regions (cbk, dbl)"]
```
Comment (benj200) [inline user/pages/03.Howtos/10.l3-high-availability/docs.en.md:30]: I'm not sure if this works well, at least in the rich diff mode, i could not see it working

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md [hunk 4]
```diff
@@ -289,6 +298,280 @@ Aug 07 09:06:18 ha-second Keepalived_vrrp[8665]: VRRP_Instance(vrrp_1) Transitio
 Aug 07 09:06:19 ha-second Keepalived_vrrp[8665]: VRRP_Instance(vrrp_1) Entering MASTER STATE
 ```
 
+[/ui-tab]
+[ui-tab title="New regions (fes, dus2)"]
```
Comment (benj200) [inline user/pages/03.Howtos/10.l3-high-availability/docs.en.md:302]: same here dus2 will come within another doc

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md [hunk 5]
```diff
@@ -289,6 +298,280 @@ Aug 07 09:06:18 ha-second Keepalived_vrrp[8665]: VRRP_Instance(vrrp_1) Transitio
...
+    }
+    virtual_ipaddress {
+        10.200.51.10
+    }
+}
+```
+
+Some of these options need special attention in cloud environments:
+
+* `garp_master_refresh 5` - forces MASTER VRRP node to generate GARP packets every 5 seconds to update ARP tables in SDN and on all VMs connected to the network.
```
Comment (benj200) [inline user/pages/03.Howtos/10.l3-high-availability/docs.en.md:415]: i see you added these options here, what about the other options from previous version ( like use_vmac and vmac_xmit_base)? They worked with cbk/dbl shouldn`t they also be mentioned somewhere?

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md (hunk 3 above)
Comment (matthiasbernhardt) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:30]: Do we still have a staging documentation system?

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md (hunk 5 above)
Comment (PioMis1984) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:415]: vmac is not working on OVN so it shouldn't be used on FES

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md (hunk 3 above)
Comment (PioMis1984) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:30]: Do you mean the tabs? They work very good. They are also used in the LBaaS page from Tutorials section:
https://docs.syseleven.de/syseleven-stack/en/tutorials/lbaas

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (PioMis1984) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:26]: Isn't that we onboard new clients on fes only and encourage existing clients to migrate from cbk and dbl to fes?
That's why I named cbk and dbl legacy ;)

[the remark this answers is not in the document]
Comment (PioMis1984) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:26]: I see that SDN backend type in regions CBK, DBL and FES is a public info and is mentioned in the Reference->Network->Known Issues page, so maybe we can use the following words:
Midonet based regions (cbk, dbl)
OVN based regions (fes)?

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md (hunk 1 above)
Comment (PioMis1984) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:9]: Will the following sentence be more accurate?

"Please consider using Octavia LBaaS solution provided by the SysEleven Stack before you decide to create a VIP VRRP cluster on your own."

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (matthiasbernhardt) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:26]: It is correct that we encourage new clients to use `fes`, but we do not discourage  to use `cbk` and `dbl` as they are still fully operational, maintained and supported. The only case when we suggest a move is when clients need a feature that we cannot provide in their current region.

As the different SDN is actually the reason why we distinguish between regions in this document, we _can_ mention that if we want, I would be fine with that.

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md (hunk 5 above)
Comment (matthiasbernhardt) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:415]: We could deliberately mark the options as `cbk and dbl only` / `fes only` in a comment to make it explicit that they haven't been forgotten by accident.

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md (hunk 3 above)
Comment (matthiasbernhardt) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:30]: It's just that the common preview utilities don't show them correctly, so we cannot review in advance when they do not get parsed due to a typo or something like that.

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md (hunk 1 above)
Comment (matthiasbernhardt) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:9]: Yes, and let's link "Octavia LBaaS" to the appropriate page in our documentation.

Comment (matthiasbernhardt) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:9]: Yes, and also link to the according page.

File: user/pages/03.Howtos/10.l3-high-availability/docs.en.md (hunk 3 above)
Comment (benj200) [reply user/pages/03.Howtos/10.l3-high-availability/docs.en.md:30]: i could view preview locally, looks fine

**Review by matthiasbernhardt — APPROVED** [review]
I'm fine with this MR now.

I think we do not cover the usage of floating IPs in conjunction with such virtual IPs yet. If we cannot add that now, we should at least create a new ticket to do it separately.

**Review by benj200 — APPROVED** [review]

**Comment by PioMis1984** [comment]
@benj200 how can I now deploy the merged change?

**Pull request merged**